src/components/Countdown.js

```diff
@@ -1,12 +1,47 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+
+const calculateTimeLeft = () => {
+  let year = new Date().getFullYear();
+  const endTime = new Date(year, 1, 21, 9, 30, 0, 0); // Feb 21, 9:30am
+  const difference = +endTime - +new Date();
+  let timeLeft = {};
+
+  if (difference > 0) {
+    timeLeft = {
+      days: Math.floor(difference / (1000 * 60 * 60 * 24)),
+      hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+      minutes: Math.floor((difference / 1000 / 60) % 60),
+      seconds: Math.floor((difference / 1000) % 60),
+    };
+  }
+
+  return timeLeft;
+};
+
+const useCounter = () => {
+  const [timeLeft, setTimeLeft] = useState(calculateTimeLeft());
+
+  useEffect(() => {
+    setTimeout(() => {
+      setTimeLeft(calculateTimeLeft());
+    }, 1000);
+  }, [timeLeft]);
+
+  return timeLeft;
+};
 
 export default function Countdown() {
+  const timeleft = useCounter();
+
   return (
     <div className="countdown">
       <div className="countdown__text-wrapper">
         <div className="countdown__text">Hacking time remaining</div>
-        <div className="font-weight-bold countdown__timer">03:34:18</div>
+        <div id="clock" className="font-weight-bold countdown__timer">
+          {timeleft.days}:{timeleft.hours}:{timeleft.minutes}:{timeleft.seconds}
```

Comment:
SHould the numbers be padded as well? So `...:4:40` looks like `...:04:40`

![Screen Shot 2021-01-16 at 2 25 20 PM](https://user-images.githubusercontent.com/22948432/104820984-abc82200-5806-11eb-86e1-6137266e9a14.png)

should be pretty easy to add by doing something like `String(n).padStart(2, "0")`